pkg/client/restclientgetter.go

```diff
@@ -51,20 +47,6 @@ func (c *restClientGetter) ToRESTConfig() (*rest.Config, error) {
 }
 
 func (c *restClientGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
```

Comment:
Nit: Rename the method to `GetDiscoveryClient` or `DiscoveryClient`? 

On that note, do we need this at all? Because we are creating a discovery client from the rest config passed to `NewActionConfigGetter` (https://github.com/operator-framework/helm-operator-plugins/pull/317/files#diff-59f6c1292b720b3b9e88d0eb2e6cf17a53d0627716cdc70dd2460227a716e907R43), and further passing it down here. This is not exported either, so looks like its not useful anyways?

Reply:
This is required to implement helm's [RESTClientGetter](https://github.com/helm/helm/blob/14d0c13e9eefff5b4a1b511cf50643529692ec94/pkg/action/action.go#L236) interface